packages/server/test/unit/plugins/child/run_require_async_child_spec.js

```diff
@@ -34,6 +34,9 @@ describe('lib/plugins/child/run_require_async_child', () => {
 
       expect(tsNodeUtil.register).to.be.calledOnce
     })
+
+    // FIXME: need to validate that TS is checked once when ts is not found as well
+    it.skip('checks for typescript only once if typescript module was not found')
```

Comment:
Is this change intentional? Not sure how it relates to the main purpose of the PR.

Reply:
Theres quite a few server/plugin/child tests that are currently skipped right now. This was a test-case I noticed that wasn't tested. Yes intentional to add & you are correct is doesn't necessarily relate to the PR changes.